### DISCUSSION TITLE
a cosa serve il pulsante Resetta le segnalazioni

### DISCUSSION THREAD

**Discussion by mfagiolihi-net** [globaleaks/globaleaks-whistleblowing-software#3997] [Q&A] [answered]
si trova in impostazioni avanzate dell'amministratore
![globaleacks2](https://github.com/globaleaks/GlobaLeaks/assets/155471806/339ad58f-3f06-4169-b4bd-02635d7c33c5)

**Comment by evilaliv3** [comment, marked as answer]
Si tratta di una funzionalità utile nel momento in cui si ha sperimentato l'uso della piattaforma e si vuole andrare in produzione resettando le segnalazioni ed i contatori. Ogni uso viene tracciato in audit log.

**Comment by VDLGEO** [comment]
Salve, ho utilizzato la funzionalità "RESETTA SEGNALAZIONI" in un ambiente di Test con l'ultima versione ma dopo il reset quando viene inserita una nuova segnalazione il sistema mostra "AL SEGNALANTE" sempre la segnalazione precedente che in teoria dovrebbe essere stata completamente cancellata. E' presente il tasto "Aggiorna" (vedi immagine allegata) all'interno della segnalazione lato "Segnalante" ed a seguito di tale operazione il tutto si allinea, ma se l'utente non fa questa operazione ha visibilità delle segnalazioni di altri. Lato Ricevente invece le informazioni sono sempre corrette.

![Immagine](https://github.com/user-attachments/assets/ee2daf90-a1c1-4eff-8a1d-d21352cdeec0)

**Comment by evilaliv3** [comment]
Buonasera,

il bottone resetta le segnalazioni cancella indelebilmente ogni segnalazione presente nel sistema e resetta i contatori.

Può spiegarmi meglio lo screenshot o provare a riprodurre la problematica che riscontra su try.globaleaks.org?

**Comment by VDLGEO** [comment]
Ciao,
provo a spiegarmi meglio riportando tutti i passaggi nel file allegato. 
[Errore.pdf](https://github.com/user-attachments/files/16894379/Errore.pdf)

**Comment by evilaliv3** [comment]
Ciao @VDLGEO ,

se vuoi raggiungermi su community.globaleaks.org ci confrontiamo ma credo trattasi di una situazione software sul client per cui il client ha memoria della propria segnalazione e ti mostra quella.
Posso assicurarti che se un amministratore cancella una segnalazione del server queste sono cancellate e sono irrecuperabili. Inoltre le segnalazioni sono cifratecon chiavi indipendenti quindi è certo che un segnalante di una segnalazione non possa accedere a dati di un'altra segnalazione.

Prova a usare tab diversi e ti renderai conto della situazione
